src/store.js

```diff
@@ -18,7 +19,7 @@ const composeEnhancers = process.env.NODE_ENV !== 'production'
     })
     : compose;
 
-const store = createStore(
+const store = createStore<State, {||}, {||}>(
```

Comment:
Will try to replace these `{||}`'s with actual types for action and dispatch in the future.